Extensions/PlatformBehavior/platformruntimebehavior.ts

```diff
@@ -229,6 +229,17 @@ namespace gdjs {
     static NORMALPLAFTORM = 0;
     static JUMPTHRU = 1;
     static LADDER = 2;
+
+    static isOnPlatformTest(
+      object1: gdjs.RuntimeObject,
+      object2: gdjs.RuntimeObject,
+      behaviorName: string
+    ): boolean {
+      const behavior1 = object1.getBehavior(
+        behaviorName
+      ) as PlatformerObjectRuntimeBehavior;
+      return behavior1.isOnFloorObject(object2);
+    }
```

Comment:
This method expose a method from PlatformerObjectRuntimeBehavior in a static context. I think it makes more sense to have it in PlatformerObjectRuntimeBehavior.
Also, I personally avoid to use "test" in a method name because it has a "unit test case" connotation and the "Is" prefix already stand for a condition.